pages/document-upload/index.tsx

```diff
@@ -0,0 +1,270 @@
+                                    Document was successfully sent to SDC.
+                                </Text>
+                            </Center>
+                            <Center>
+                                <Text
+                                    fontWeight="200"
+                                    fontSize="15px"
+                                    marginBottom="20px"
+                                    textAlign={["center"]}
+                                >
+                                    Keep and eye out on the status of your
+                                    background check within the next few weeks.
+                                </Text>
+                            </Center>
+                            <Center>
+                                <Button
+                                    backgroundColor="white"
+                                    borderColor="brand.400"
+                                    width="366px"
+                                    height="54px"
+                                    fontSize="16px"
+                                    fontWeight="400"
+                                    color="#0C53A0"
+                                    border="2px"
+                                    mt="20px"
+                                    onClick={() =>
+                                        router.push("/myaccounts").then(() => {
+                                            window.scrollTo({ top: 0 });
+                                        })
+                                    }
+                                >
+                                    View Account
+                                </Button>
+                            </Center>
+                            <Center>
+                                <Button
+                                    backgroundColor="#0C53A0"
+                                    borderColor="brand.400"
+                                    width="366px"
+                                    height="54px"
+                                    fontSize="16px"
+                                    fontWeight="400"
+                                    color="white"
+                                    border="1px"
+                                    mt="20px"
+                                    onClick={() =>
+                                        router.push("/").then(() => {
+                                            window.scrollTo({ top: 0 });
+                                        })
+                                    }
+                                >
+                                    Browse programs
+                                </Button>
+                            </Center>
+                        </Box>
+                    </Center>
+                </VStack>
+            </Wrapper>
+        );
+    };
+    if (uploadSuccess) {
+        return uploadSuccessUI();
+    } else {
+        return uploadDocumentUI();
```

Comment:
I would try to put these into two pieces into separate components to make this file smaller. Or we can refactor this when adding state management later on.

Reply:
I'll leave this for refactoring since if I were to separate these into components, I'm sure if they are actually reusable in other pages.